packages/atomic-react/README.md

```diff
@@ -265,3 +272,133 @@ import {AtomicSearchInterface} from '@coveo/atomic-react';
+
+Then, the same need to be done for the matching version of `@coveo/headless` used by the matching `@coveo/atomic` version.
+
+Using the above example with `@coveo/atomic@1.108.2`:
+
+```bash
+npm view @coveo/atomic@1.108.2
+
+
+@coveo/atomic@1.108.2 | Apache-2.0 | deps: 15 | versions: 728
+A web-component library for building modern UIs interfacing with the Coveo platform
+https://docs.coveo.com/en/atomic/latest/
+
+dist
+.tarball: https://registry.npmjs.org/@coveo/atomic/-/atomic-1.108.2.tgz
+.shasum: 6e844073f55f10328b4b828c0190b8fe31403fbf
+.integrity: sha512-LhFR6k7NdGi8pYHmWOq3gM3rE0zeTKc6biTa1Z+ZKr5TEXIr97rpxi1HSnViQUowQYYCcY44dvsAYz2yyPvXTA==
+.unpackedSize: 87.1 MB
+
+dependencies:
+@coveo/bueno: 0.42.1                  @stencil/store: 1.5.0                 focus-visible: 5.2.0
+@coveo/headless: 1.103.3              coveo-styleguide: 9.34.4              i18next-http-backend: 1.4.1
+@popperjs/core: ^2.11.6               dayjs: 1.11.5                         i18next: 20.6.1
+@salesforce-ux/design-system: ^2.16.1 dompurify: 2.3.10                     stencil-inline-svg: 1.1.0
+@stencil/core: 2.17.3                 escape-html: 1.0.3                    ts-debounce: ^4.0.0
+
+```
+
+In the above example, we can see that that `@coveo/atomic@1.108.2` has a dependency on `@coveo/headless@1.103.3`.
+
+In summary, as of the time of this writing, the versions that we should keep in mind are the following:
+
+- `@coveo/atomic-react` -> 1.23.11
+- `@coveo/atomic` -> 1.108.2
+- `@coveo/headless` -> 1.103.3
+
+```html
+<head>
+  <!-- React and ReactDOM need to be included  -->
+  <script crossorigin src="https://unpkg.com/react@18/umd/react.production.min.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@18/umd/react-dom.production.min.js"></script>
+
+  <!-- Optional script, which allows to use JSX directly in an inline script in the page -->
+  <script crossorigin src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
+
+  <!-- @coveo/headless need to be included as a dependency -->
+  <!-- Note the matching major and minor version as explained above -->
+  <script crossorigin src="https://static.cloud.coveo.com/headless/v1.103/headless.js"></script>
+
+  
+  <!-- @coveo/atomic need to be included as a dependency -->
+  <!-- Note the matching major and minor version as explained above -->
+  <script crossorigin type="module" src="https://static.cloud.coveo.com/atomic/v1.108/atomic.esm.js"></script>
+  
+  <!--And then finally @coveo/atomic-react is included -->
+  <!-- Note the matching major and minor version as explained above -->
+  <script crossorigin src="https://static.cloud.coveo.com/atomic-react/v1.23/iife/atomic-react.min.js"></script>
+</head>
+```
+
+Once this is done, you can now start using `CoveoAtomicReact` directly with an inline script tag: 
```

Comment:
```suggestion
Once this is done, you can start using `CoveoAtomicReact` directly with an inline script tag: 
```
(since you already set a time frame with `Once this is done`)